recipe/meta.yaml

```diff
@@ -1,34 +1,43 @@
-{% set version = "5.2.0" %}
-{% set datumgrid_ver = "1.7" %}
+{% set version = "6.0.0" %}
+{% set datumgrid_ver = "1.8" %}
```

Comment:
We probably need to split the `datumgrid` part into its own package, but let's do that in another PR.

Reply:
Sounds like a good idea 👍